manifests/base/monitoring/recording-rules.yaml

```
@@ -0,0 +1,118 @@
+        component: "training-operator"
+        
+    # Rule 2: Active training jobs by framework
+    # Used for: Current cluster load monitoring
+    - expr: |
+        sum by (framework) (
+          training_operator_jobs_created_total - training_operator_jobs_deleted_total
+        )
+      record: training_operator:jobs_active_by_framework:sum
+      labels:
+        component: "training-operator"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)